### ISSUE TITLE
Add the ability to record and reply a pulse stream

### ISSUE THREAD

**Issue by sdwilsh** [sdwilsh/node-pulse#6]
Request from @robarnold, but I'm hashing out the details.  Debugging things is hard, and it'd be nice to be able to say something like `connection.startRecording('filename')` and then `connection.stopRecording()` to record a stream of events.  Then you could just `pulse.newRecordedConnection("filename", topics)` and have the events streamed to you.

Not sure if we would want to delay the recording to be the same time or not.  Does @LegNeato have any thoughts on this?

**Comment by LegNeato** [comment]
Yeah, not sure, I guess you could make it configurable (some use cases may care about the timing between, some may just use it to inform state). Also not sure if you'd want to rewrite the timestamps or not (probably make that configurable too)

FYI, eventually I am going to have all pulse messages going into a riak exchange so you can query riak historically for any message or message stream. But that's pretty far down the road...
